docs/automated-process.md

```diff
@@ -11,6 +11,29 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Membership to [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
```

Comment:
Since the other two points are formulated as actions, this one should probably be one as well:
```suggestion
- Become a member of the [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
```